code/modules/projectiles/guns/projectile/pistol.dm

```diff
@@ -298,4 +298,83 @@
 	if(loaded.len)
 		icon_state = "leyon"
 	else
-		icon_state = "leyon-e"
\ No newline at end of file
+		icon_state = "leyon-e"
+
+
+/obj/item/weapon/gun/projectile/papadov
+	name = ".380 pistol"
+	desc = "A pistol fit to carry to the opera or for Sol agents with a licence to kill."
+	icon_state = "papadov"
+	item_state = null
+	w_class = 1
+	caliber = ".380"
+	silenced = 0
+	origin_tech = list(TECH_COMBAT = 2, TECH_MATERIAL = 2, TECH_ILLEGAL = 4)
+	fire_sound = 'sound/weapons/gunshot_pistol.ogg'
+	load_method = MAGAZINE
+	allowed_magazines = list(/obj/item/ammo_magazine/c380m)
+	magazine_type = /obj/item/ammo_magazine/c380m/t
+
+/obj/item/weapon/gun/projectile/papadov/attack_hand(mob/user as mob)
+	if(user.get_inactive_hand() == src)

```

Comment:
You should use `&&` to lessen the pyramid of doom. Or early returns.

So either:
```dm
	if (user.get_inactive_hand() == src && silenced)
		// stuff here

	..()
```